lib/sms.js

```diff
@@ -0,0 +1,84 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+var Nexmo = require('nexmo')
+var P = require('bluebird')
+
+module.exports = function (log, translator, templates, config) {
+  var nexmo = new Nexmo({
+    apiKey: config.apiKey,
+    apiSecret: config.apiSecret
+  })
+  var sendSms = P.promisify(nexmo.message.sendSms, { context: nexmo.message })
+
+  var LINK = 'http://mzl.la/1HOd4ec'
```

Comment:
should we make this part of config? (just in case we need to tweak this, change the page and redeploy)

Also, can we use HTTPS for this? :)